monkey/monkey_island/cc/services/agent_binary_service/i_agent_binary_service.py

```diff
@@ -1,12 +1,28 @@
 import abc
-from typing import BinaryIO
+from typing import BinaryIO, Optional
 
 
 class IAgentBinaryService(metaclass=abc.ABCMeta):
     """
     A service for retrieving and manipulating agent binaries
     """
 
+    @abc.abstractmethod
+    def get_masque(self) -> Optional[str]:
+        """
+        Get the masque to append to the Agent binary
+
+        :return: The value of the masque
+        """
+
+    @abc.abstractmethod
+    def set_masque(self, masque: Optional[str]):
+        """
+        Set the masque to append to the Agent binary
+
+        :param masque: The value to set the masque to
+        """
```

Comment:
Avoid the word "append" when discussing `IAgentBinaryService`. This service should have no knowledge of specifically how the masque is applied, as this could change independently of the service.

Similarly, we should avoid this information in the documentation or UI. The only thing that needs to know the masque is appended and not added in some other way is the masquerade decorator.